conf/modules.config

```diff
@@ -465,6 +470,7 @@ process {
     }
 
     withName: 'SAMTOOLS_STATS' {
+        ext.prefix       = { "${meta.id}.cram" }
```

Comment:
I think this is already in another PR, Susis maybe 🤔

Reply:
Need it there for the tests anyway

Reply:
ah ok :) makes sense